_maps/map_files/Aetherwhisp/Aetherwhisp.dmm

```diff
@@ -2671,6 +2670,7 @@
 "lgK" = (/obj/machinery/door/firedoor,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/machinery/door/airlock/glass_command{name = "Bridge"; req_access_txt = "0"; req_one_access_txt = "19;35"},/obj/structure/disposalpipe/segment{dir = 4},/turf/open/floor/plasteel/black,/area/shuttle/ftl/bridge/meeting_room)
 "lqj" = (/obj/machinery/power/apc/ship{auto_name = 1; dir = 1; name = "north bump"; pixel_y = 24},/obj/structure/cable{d2 = 8; icon_state = "0-8"},/obj/structure/cable{crit_fail = 0; d1 = 4; d2 = 8; icon_state = "4-8"},/obj/machinery/camera{c_tag = "FTL Drive"; dir = 2; network = list("SS13","Supermatter")},/turf/open/floor/engine,/area/shuttle/ftl/engine/engineering{icon_state = "engine"; name = "FTL Drive"})
 "lxj" = (/obj/structure/closet/emcloset,/turf/open/floor/plasteel/blue/side{dir = 4},/area/shuttle/ftl/subshuttle/pod_3)
+"lzK" = (/obj/machinery/mass_driver{dir = 8; id = "supermatter_eject"; power = 4},/obj/machinery/power/supermatter_shard/crystal{anchored = 0},/turf/open/floor/engine/airless,/area/shuttle/ftl/engine/supermatter_core)
```

Comment:
Just remove the `{anchored = 0}` entirely so it changes with future code if need be.